### PR TITLE
Fix namespace pick when trying to connect without permissions on `%SYS`

### LINKED ISSUE #1096: No connection with roleset that can't see %SYS
When trying to connect to a server with a user that does not have the privileges required to access, or even "see", the %SYS namespace, connection fails **immediately**, with an **instant** error message 

`This server does not have specified namespace '%SYS'. You must select one of the following: USER.`

Usually, with user that are able to see and use %SYS, we get a list of namespaces available to connect to, with the problematic case it goes straight to the error message and does not provide a dropdown list. %SYS was never specified in server settings or during attempts to connect.

### PR DESCRIPTION
**Pull request by isc-bsaviano**
This PR fixes #1096

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -439,12 +439,12 @@ export class AtelierAPI {
     }
   }
 
-  public serverInfo(): Promise<Atelier.Response<Atelier.Content<Atelier.ServerInfo>>> {
+  public serverInfo(checkNs = true): Promise<Atelier.Response<Atelier.Content<Atelier.ServerInfo>>> {
```
Comment (gjsjohnmurray) [inline src/api/index.ts:442]:
```suggestion
  public serverInfo(checkNs: boolean = true): Promise<Atelier.Response<Atelier.Content<Atelier.ServerInfo>>> {
```

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
Just one minor suggestion added.

File: src/api/index.ts (hunk 1 above)
Comment (isc-bsaviano) [reply src/api/index.ts:442]: ESLint strips that type annotation out when you run `mpn run lint-fix`.

Comment (gjsjohnmurray) [reply src/api/index.ts:442]: Fair enough.

**Review by gjsjohnmurray — APPROVED** [review]

**Pull request merged**